### PR TITLE
GA release notes

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `prerelease package`.  
 
+## 1.0.627-prerelease
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:23]: Add a comment here to remember to update the prerelease build number.

File: microsoft-edge/webview2/releasenotes.md [hunk 2]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `prerelease package`.  
 
+## 1.0.627-prerelease  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.672-prerelease] \| minimum Microsoft Edge version 88.0.672.0.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:27]: @liminzhu Please advise what minimum version we should put here.

File: microsoft-edge/webview2/releasenotes.md [hunk 3]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+*    Updated [WebResourceResponseReceived][ReferenceWin3210672Icorewebview2experimentalAddWebresourceresponsereceived] event including adding immutable [WebResourceResponseView][ReferenceWin3210672Icorewebview2experimentalwebresourceresponseview] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin3209628Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent] becoming [WebResourceResponseView::GetContent][ReferenceWin3210672Icorewebview2experimentalwebresourceresponseviewGetcontent].
+*    Disabled [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+* Added [SystemCursorId](link to ICoreWebView2Controller.SystemCursorId) for Visual Hosting.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:38]: TODO: add links to reference docs when they get generated

Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md:38]:
```suggestion
*    Added [SystemCursorId](link to ICoreWebView2Controller.SystemCursorId) for Visual Hosting.
```

File: microsoft-edge/webview2/releasenotes.md [hunk 4]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+* Bug fixes for IME issues in Visual Hosting.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:39]: Can you elaborate what issues are fixed? Maybe say IME is now supported in visual hosting?

File: microsoft-edge/webview2/releasenotes.md [hunk 5]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+
+#### .NET  
+
+* Updated ICoreWebView2 to expose environment variable.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:43]: There is no ICoreWebView2 interface in .NET; it should be CoreWebView2 class

File: microsoft-edge/webview2/releasenotes.md [hunk 6]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+* Changed implementations of custom EventArgs classes in `Microsoft.Web.WebView2.Core` namespace to be subclasses of `System.EventArgs` or `System.CancelEventArgs`. \([\#250][GithubMicrosoftedgeWebviewfeedbackIssue250]\).  
+* Added support for CoreWebView2CreationProperties in WinForms.
+* Added .NET WebResourceRequested APIs.  \([\#219][GithubMicrosoftedgeWebviewfeedbackIssue219]\).  
+* Updated WinForms Designer Source property to default/reset to null. \([\#177][GithubMicrosoftedgeWebviewfeedbackIssue177]\).  
+* Updated WebView2 bounds in WebView2.Init() to support non-100% DPI modes.  \([\#432][GithubMicrosoftedgeWebviewfeedbackIssue432]\).  
+* Updated BuildWindowCore and DestroyWindowCore to be more robust.  \([\#382][GithubMicrosoftedgeWebviewfeedbackIssue382]\).  
+* Updated .NET Loader base to load on process bit instead of OS arch. \([\#431][GithubMicrosoftedgeWebviewfeedbackIssue431]\).  
+* Added CreationProperties support to WinForms?? \([\#204][GithubMicrosoftedgeWebviewfeedbackIssue204]\).  
+ 
+## 1.0.622
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:53]:
```suggestion
## 1.0.622.22  
```

**Review by peiche-jessica — DISMISSED** [review]

File: microsoft-edge/webview2/releasenotes.md [hunk 7]
```diff
@@ -20,6 +20,49 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+*   Updated the [WebResourceResponseReceived][ReferenceWin32Icorewebview2experimentalAddWebresourceresponsereceived10674] event, and added immutable [WebResourceResponseView][ReferenceWin32Icorewebview2experimentalwebresourceresponseview10674] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin32Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent09628] to become [WebResourceResponseView::GetContent][ReferenceWin32Icorewebview2experimentalwebresourceresponseviewGetcontent10674].
+*   Turned off [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+*   Added [SystemCursorId][ReferenceWin32Icorewebview2experimentalcompositioncontroller2GetSystemcursorid10674] for Visual Hosting.
+*   Bug fixes for IME issues in Visual Hosting.
+
+#### .NET  
+
+*   Updated [CoreWebView2][DotnetApiMicrosoftWebWebview2CoreCorewebview2] class to expose environment variables.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/releasenotes.md:43]: Did you mean to say to we now expose the CoreWebView2Environment that is used to create the associated CoreWebView2 class via CoreWebView2.Environment property? Or you are actually referring to system/user environment variables?

File: microsoft-edge/webview2/releasenotes.md (hunk 4 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md]: @liminzhu - would "Input Method Header is now supported in visual hosting" work?

File: microsoft-edge/webview2/releasenotes.md (hunk 7 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/releasenotes.md:43]: Following up w/ @pagoe-msft

File: microsoft-edge/webview2/releasenotes.md [hunk 8]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+## 1.0.622.22  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.622.22] \| minimum WebView2 Runtime version 86.0.622.22.  
+
+#### General  
+
+> [!IMPORTANT]
+> **Announcement**: This release is now Generally Available (GA).
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:62]: > This release is now Generally Available (GA).
Recommend `Win32 C/C++ WebView2 is now Generally Available (GA). Starting with this release, Release SDKs will be forward-compatible. See [GA announcement blog post]() for more details.`

I don't have the link to the post before it goes live, but you can make an aka.ms link here and redirect to the blog once it's live. Let @peiche-jessica  or me know if you don't know how to do that.

Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md:62]: Something like `aka.ms/wv2gablogpost`? Where should I point it to?

File: microsoft-edge/webview2/releasenotes.md [hunk 9]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+*   Updated the [WebResourceResponseReceived][ReferenceWin32Icorewebview2experimentalAddWebresourceresponsereceived10674] event, and added immutable [WebResourceResponseView][ReferenceWin32Icorewebview2experimentalwebresourceresponseview10674] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin32Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent09628] to become [WebResourceResponseView::GetContent][ReferenceWin32Icorewebview2experimentalwebresourceresponseviewGetcontent10674].
+*   Turned off [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+*   Added [SystemCursorId][ReferenceWin32Icorewebview2experimentalcompositioncontroller2GetSystemcursorid10674] for Visual Hosting.
+*   Input Method Header is now supported in visual hosting.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:39]: > *   Input Method Header is now supported in visual hosting.

I think you're referring to the IME bugs we fixed? I'd just say something like `Added bug fixed for Input Method in Visual Hosting`.

File: microsoft-edge/webview2/releasenotes.md (hunk 7 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/releasenotes.md]: variables -> property (singular)?

File: microsoft-edge/webview2/releasenotes.md [hunk 10]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
 
+## 1.0.674-prerelease  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.674-prerelease] \| minimum Microsoft Edge version 88.0.674.0.
+
+#### General
+
+*   Added [WebView2 Group Policies][DeployedgeMicrosoftEdgeWebviewPolicies]. For more information on best practices, navigate to [group policies for WebView2][Webview2ConceptsEnterpriseGroupPoliciesForWebview2].
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:31]: We decided on Friday to break and change the naming of some policies, so let's delete the group policy entry in release notes for now.

File: microsoft-edge/webview2/releasenotes.md [hunk 11]
```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
...
+*   Added [NavigateWithWebResourceRequest][ReferenceWin32Icorewebview2experimentalNavigatewithwebresourcerequest10674] method that lets you provide post data or additional request headers during navigation.
+*   Added [DOMContentLoaded][ReferenceWin32Icorewebview2experimentalAddDomcontentloaded10674] event that runs when the initial HTML document is parsed.
```
Comment (liminzhu) [inline microsoft-edge/webview2/releasenotes.md:33]: > parsed

loaded and parsed

File: microsoft-edge/webview2/releasenotes.md (hunk 8 above)
Comment (liminzhu) [reply microsoft-edge/webview2/releasenotes.md:62]: https://blogs.windows.com/msedgedev/ until the blog is live

File: microsoft-edge/webview2/releasenotes.md (hunk 4 above)
Comment (liminzhu) [reply microsoft-edge/webview2/releasenotes.md]: A couple odd ones that are probably not worth elaborating here.

**Review by peiche-jessica — COMMENTED** [review]
Unblocking the PR

**Review by Reezaali — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**